Image_process/processImage.py

```diff
@@ -0,0 +1,42 @@
+from PIL import Image
+
+
+class ImageProcess:
+    def __init__(self, image_path):
+        self.image_path = image_path
+
+    def resize_image(self):
+        """
+        Resize the image to the specified width and height.
+        :param width: Target width of the image.
+        :param height: Target height of the image.
+        :return: Resized Image object.
+        """
+        width = 128
+        height = 128
+        #if image isn't square raise a warning and resize to square
+        image = Image.open(self.image_path)
+        if image.size[0] != image.size[1]:
+            print("Warning: Image is not square. Resizing to square.")
+
+        return image.resize((width, height))
+    
+    def save_image(self, image, save_path):
+        """
+        Save the image to the specified path.
+        :param image: Image object to save.
+        :param save_path: File path to save the image.
+        """
+        #make sure file type is a jpg if not make it a jpg
+        if not save_path.endswith(".jpg"):
```

Comment:
should not be hardcoded with `jpg`, what if we wanted to support `png`'s or `gif`'s?

Reply:
I haven't though of gif's I feel like stationary images should be a jpg because we don't need to focus too much on high quality images for the scope of this project